spec/spec_helper.rb

```diff
@@ -5,25 +5,19 @@
 
 CENTOS_7 = {
   platform: 'centos',
-  version: '7.2.1511'
-}.freeze
-
-CENTOS_6 = {
-  platform: 'centos',
-  version: '6.7'
-}.freeze
-
-DEBIAN_8 = {
-  platform: 'debian',
-  version: '8.4'
+  version: '7.2.1511',
```

Comment:
We need to keep all of these platforms (including Debian for workstations). It looks like we need to switch the workstations over to this cookbook and remove the references we have in the role eventually.

Reply:
Ahh, I was forgetting why we had Debian. :P